package.json

```diff
@@ -17,10 +17,10 @@
     "pretest:integration": "./scripts/install-npm-packages-for-integration-tests",
     "test:integration": "cross-env NODE_ENV=test jest --config integration-tests/.jestrc.json",
     "format": "prettier --write '{packages,integration-tests,types}/**/*.js'",
-    "coverage": "npm run build && npm test -- --no-watchman --coverage",
-    "coverage:ci": "npm test -- --no-watchman --coverage && codecov",
-    "semantic-release": "lerna-semantic-release pre && lerna-semantic-release post && lerna-semantic-release perform",
-    "manual-release": "./scripts/publish-manually",
+    "test:coverage": "npm run build && npm test -- --no-watchman --coverage",
```

Comment:
Renaming these makes sense to me as they're nicely grouped then when e.g. running an `npm run`.

Reply:
Same here: shouldn't we use `yarn` consistently?